packages/zanuda/zanuda.2.0.0/opam

```diff
@@ -0,0 +1,59 @@
+opam-version: "2.0"
+synopsis: "Linter for OCaml+dune projects"
+description:
+  "Lints for OCaml projects. Primary usage is for teaching/education"
+maintainer: ["kakadu@pm.me"]
+authors: ["Kakadu"]
+license: "LGPL-3.0-only"
+tags: ["lint" "test"]
+homepage: "https://github.com/Kakadu/zanuda"
+bug-reports: "https://github.com/Kakadu/zanuda/issues"
+depends: [
+  "dune" {>= "3.4"}
+  "ocaml" {= "4.14.2" | >= "5.3.0" & < "5.4.0"}
+  "yojson" {>= "2.0.0"}
+  "angstrom" {>= "0.15.0" & <= "0.16.0"}
+  "sexplib"
+  "bisect_ppx"
+  "dune-build-info"
+  "ppx_assert"
+  "ppx_expect_nobase"
+  "ppx_optcomp"
+  "base" {with-test}
+  "ppx_assert" {with-test}
+  "ppx_inline_test" {with-test}
+  "ppx_expect" {with-test}
+  "ppx_fields_conv" {with-test}
+  "ppx_deriving" {with-test}
+  "ppx_blob" {with-test}
+  "menhir" {with-test}
+  "ocamlformat" {= "0.27.0" & with-dev-setup}
+  "odoc" {with-doc}
+  "odig" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/Kakadu/zanuda.git"
+# We have issues with tests on these distributions. TODO: Fix later
+available: os-family != "alpine" & os-family != "opensuse"
```

Comment:
```suggestion
```
What's the issue with opensuse requiring it to be marked unavailable?
In 32b6354 and https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/32b6354188d1e3f1340a0327dab79985fa4a6f57
it both compiled and passed the test suite, AFAICS.